testpilot/frontend/static-src/app/main.js

```diff
@@ -42,39 +42,62 @@ app.extend({
     app.me = new Me();
     app.experiments = new ExperimentsCollection();
 
-    Promise.all([
-      app.me.fetch(),
-      app.experiments.fetch()
-    ]).then(() => {
-      app.me.updateEnabledExperiments(app.experiments);
+    app.pageManager = new PageManager({
+      pageContainer: document.querySelector('[data-hook=page-container]')
+    });
 
-      app.pageManager = new PageManager({
-        pageContainer: document.querySelector('[data-hook=page-container]')
-      });
+    if (!app.router.history.started()) {
```

Comment:
Also, moving page manager & router init outside of the experiments fetch means we can show the error page. Side effect will be that the page will render in with experiments until the API actually returns results, so usually a sub-second lag between the bulk of the page rendering and then filling in with experiments. Previously, the whole page would just stay white until the experiments came in.